app/config/colors.js

```diff
@@ -2,10 +2,12 @@ const colors = {
   LF_DARK_GRREEN: '#17a555',
   LF_LIGHT_GREEN: '#6cc067',
   GRAY: '#6e6f72',
+  MID_GRAY: '#9E9E9E',
   LIGHT_GRAY: '#c8c8c8',
```

Comment:
@kusalshrestha it's better to follow single style for naming
`LF_DARK_GRREEN: '#17a555',`
I prefer to follow this
 `MID_GRAY: '#9E9E9E'`

Reply:
🆗